notebooks/tutorial/8 - Getting tomes from the ADX.ipynb

```diff
@@ -0,0 +1,230 @@
+    }
+   },
+   "cell_type": "markdown",
+   "id": "9ac3586b",
+   "metadata": {},
+   "source": [
+    "![adx_tome_explainer.png](attachment:adx_tome_explainer.png)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "9db8f974",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "dataset_id = \"c5943d0f4bbce90b2f7e5f8488595e86\"\n",
+    "bucket = None \n",
+    "prefix = None"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "2e805cf6",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "if dataset_id == \"c5943d0f4bbce90b2f7e5f8488595e86\":\n",
+    "    raise Exception(\"Your dataset_id is still the default. Please subscribe on the ADX\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "f6715957",
+   "metadata": {},
+   "source": [
+    "# Download the tomes\n",
+    "\n",
+    "Since the tomes are only about 15 GB the cost is very little to transfer all of them. Therefore we will simply transfer them all. Check the [previous notebook]() for a cost calculator."
+   ]
+  },
+  {
```

Comment:
Last step is to give them the aws sync command for the tomes and let them know they can now load them by name like any other tome.

Reply:
gotit. maybe a few examples for good measure